src/test/java/GitHubIssueCreationAndCloseTest.java

```diff
@@ -0,0 +1,31 @@
+import GitHubIssueCreationAndClose.HomePage;
+import GitHubIssueCreationAndClose.IssuesPage;
+import GitHubIssueCreationAndClose.MainPage;
+import GitHubIssueCreationAndClose.RepositoryPage;
+import Helpers.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static Helpers.ColorPrinter.printColorMessage;
+
+public class GitHubIssueCreationAndCloseTest extends BaseTest {
+
+    Logger logger = LogManager.getLogger();
+
+    @Test
+    public void issueCreationAndCloseBaseTest() {
```

Comment:
it's a good idea to separate this test for 2 - an issue creation and an issue closing